GVFS/GVFS.FunctionalTests/Categories.cs

```diff
@@ -6,13 +6,17 @@ public static class Categories
         public const string FastFetch = "FastFetch";
         public const string GitCommands = "GitCommands";
 
-        public const string Windows = "Windows";
+        public const string WindowsOnly = "WindowsOnly";
+        public const string MacOnly = "MacOnly";
 
-        public static class Mac
+        public static class MacTODO
         {
-            public const string M1 = "M1_CloneAndMount";
+            // The FailsOnBuildAgent category is for tests that pass on dev
+            // machines but not on the build agents
+            public const string FailsOnBuildAgent = "FailsOnBuildAgent";

```

Comment:
I'm just curious, any theories as to why some of the tests are failing on the build agent?

Reply:
@nickgra had investigated this a bit.  The failures all have to do with non-ASCII file names, and I think Nick mentioned there must be a setting on the build agents that impacts how the file names are displayed.  

From what I remember it was something to do with continuation characters being added into the names, @nickgra, do you remember any of the details?

Reply:
You've nailed it, that's about as far as I got in the investigation. I have some ideas on how we can debug this further, I'll follow up offline.